### PR TITLE
Feat/spanish es

### PR DESCRIPTION
**Pull request by MohamedHamouGisaia**

### CODE REVIEW

**Review by sebbousquet — APPROVED** [review]

File: scripts/start.sh [hunk 1]
```diff
@@ -80,6 +80,18 @@ else
   fetchI18nFRContent;
 fi
 
+
+fetchI18nESContent(){
+  echo "Download the fr.json file from \"${ARLAS_WUI_I18N_ES_URL}\" ..."
```
Comment (QuCMGisaia) [inline scripts/start.sh:85]: download the es json

File: src/assets/i18n/es.json [hunk 2]
```diff
@@ -0,0 +1,108 @@
+{
+	"Map Settings": "Configuración de la mapa",
+	"Basemaps": "Mapas base",
+	"Geo-filter tools": "Geo-filtros",
+	"Search a place": "",
```
Comment (QuCMGisaia) [inline src/assets/i18n/es.json]: There is no translation here, is it normal ?

File: src/assets/i18n/fr.json [hunk 3]
```diff
@@ -19,14 +19,14 @@
 	"Error while retrieving dashboards": "Erreur pendant la récupération des dashboards",
 	"Find a place": "Rechercher un lieu",
 	"No results": "Pas de résultat",
-	"Unable to access find location service": "Impossible d'accéder au service de geocoding",
+	"Unable to access find location service": "No se puede acceder al servicio de geocodificación",
```
Comment (QuCMGisaia) [inline src/assets/i18n/fr.json:22]: Should be in French

File: src/assets/tour/tour_es.json [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+        "content": " <strong>ACCIÓN :</strong> Haga clic en el botón 'Configuración del mapa' (icono de lupa)",
+        "placement": "left",
+        "target": ".arlas-map-settings-container",
+        "yOffset": -15,
+        "showNextButton": false,
+        "nextOnTargetClick": true
+      },
+      {
+        "title": "Aplicar los cambios",
+        "content": "<br> - Elige cómo buscar datos geográficamente (pestaña Peticiones geográficas). <br><br>  <strong>ACTION :</strong> Haga clic para aplicar los cambios.",
```
Comment (QuCMGisaia) [inline src/assets/tour/tour_es.json:53]: ACTION is used here, and above ACCION is used

**Review by QuCMGisaia — APPROVED** [review]

File: src/assets/tour/tour_es.json (hunk 4 above)
Comment (MohamedHamouGisaia) [reply src/assets/tour/tour_es.json]: fixed

File: src/assets/i18n/fr.json (hunk 3 above)
Comment (MohamedHamouGisaia) [reply src/assets/i18n/fr.json]: fixed

**Pull request merged**